docker/test/gen-certs/mongodb-test-ca.crt

```diff
@@ -0,0 +1,39 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Do we really have to keep all those files in git? Can we generate them on depend with gen-certs.sh or (preferably) with mkcert?

Reply:
certs generated with mkcert are not working. It seems it is because mkcert is not setting the DNS for ip addresses. Added auto generation with the current script